Source/processcontainers/implementations/DobbyImplementation/DobbyImplementation.cpp

```diff
@@ -0,0 +1,291 @@
+            std::string containerInfoString = admin.mDobbyProxy->getContainerInfo(_descriptor);
+
+            if (containerInfoString.empty())
+            {
+                TRACE_L1("Failed to get info for container %s", _name.c_str());
+            }else{
+                // Dobby returns the container info as JSON, so parse it
+                JsonObject containerInfoJson;
+                WPEFramework::Core::OptionalType<WPEFramework::Core::JSON::Error> error;
+                if (!WPEFramework::Core::JSON::IElement::FromString(containerInfoString, containerInfoJson, error)) {
+                    TRACE_L1("Failed to parse Dobby Spec JSON due to: %s", WPEFramework::Core::JSON::ErrorDisplayMessage(error).c_str());
+                }
+
+                JsonArray pids = containerInfoJson["pids"].Array();
+
+                if(pids.Length() > 0) {
+                    // first PID is the DobbyInit process which launches the container
+                    returnedPid = pids[0].Number();
+                    TRACE_L1("%s container PID is: %d", _name.c_str(), returnedPid);
+
+                    // remember PID so it doesn't need to be read again
+                    _pid = returnedPid;
+                }
+            }
+        }
+        else
+        {
+            // Value was already read before, use this value
+            returnedPid = _pid;
+        }
+        return returnedPid;
+    }
+
+    bool DobbyContainer::IsRunning() const
+    {
+        bool result = false;
+        auto& admin = static_cast<DobbyContainerAdministrator&>(DobbyContainerAdministrator::Instance());
+
+        // We got a state back successfully, work out what that means in English
+        switch (static_cast<IDobbyProxyEvents::ContainerState>(admin.mDobbyProxy->getContainerState(_descriptor)))
+        {
+        case IDobbyProxyEvents::ContainerState::Invalid:
+            result = false;
+            break;
+        case IDobbyProxyEvents::ContainerState::Starting:
+            result = false;
+            break;
+        case IDobbyProxyEvents::ContainerState::Running:
+            result = true;
+            break;
+        case IDobbyProxyEvents::ContainerState::Stopping:
+            result = true;
+            break;
+        case IDobbyProxyEvents::ContainerState::Paused:
+            result = true;
+            break;
+        case IDobbyProxyEvents::ContainerState::Stopped:
+            result = false;
+            break;
+        default:
+            result = false;
+            break;
+        }
+
+        return result;
+    }
+
+    bool DobbyContainer::Start(const string& command, IStringIterator& parameters)
+    {
+        bool result = false;
+        auto& admin = static_cast<DobbyContainerAdministrator&>(DobbyContainerAdministrator::Instance());
+
+        std::list<int> emptyList;
+
+        // construct the full command to run with all the arguments
+        std::string fullCommand = "/usr/bin/" + command;
```

Comment:
I think we can remove "/usr/bin" prefix and rely on PATH

Reply:
If you are concerned about the usability outside of the Sky domain, please also not line 67 in this file which has hardcoded values.

Reply:
@pwielders I think we are OK with the line 67 hard coded values for now. 
Can we merge this PR soon ? thanks!